go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	go.etcd.io/etcd/etcdutl/v3 v3.6.0-alpha.0
 	go.etcd.io/etcd/pkg/v3 v3.6.0-alpha.0
 	go.etcd.io/etcd/server/v3 v3.6.0-alpha.0
-	go.etcd.io/etcd/tests/v3 v3.0.0-00010101000000-000000000000
+	go.etcd.io/etcd/tests/v3 v3.5.17
```

Comment:
Don't think changes to `go.mod` nor `go.sum` are needed. As `porcupine` is a test dependency it's not mentioned in `go.mod`. We just need to update `tests/go.mod` and `tests/go.sum`